### ISSUE TITLE
Erro SAM D21

### ISSUE THREAD

**Issue by zezao** [udev-br/SIGFOX-EDK#2]
Estou obtendo erros na parte dos arquivos relacionados ao processador SAM D21, mesmo após ter instalado todos os pacotes que dizem respeito ao mesmo.

![Sem título 2](https://user-images.githubusercontent.com/17934832/54866274-72304800-4d50-11e9-8e52-9d86eb31eb76.png)

![Sem título](https://user-images.githubusercontent.com/17934832/54866276-7b211980-4d50-11e9-98b7-7e30eba5cb0b.png)

Alguma ideia do que pode estar acontecendo?

**Comment by eliveltonpsantos** [comment]
uDEV's Template isn't working. To work around this problem, create a new GCC C Executable Project, import libudev_sigfox_edk.a as manual describe and create manually all folders and files. Don't forget to include createds folder at Project Proprieties -> Toolchain -> ARM/GNU C Compiler -> Include Paths.

**Comment by zezao** [comment]
I figured out the problem. When installing the packages for the SAMD21, the version 1.2.280 wasn't included in the Device Pack Manager list, which was the version used on the uDEV SIGFOX EDK, as you can see below:

![54866276-7b211980-4d50-11e9-98b7-7e30eba5cb0b (1)](https://user-images.githubusercontent.com/17934832/56214613-9839c700-6034-11e9-96c2-900864531e3b.png)


![Sem título](https://user-images.githubusercontent.com/17934832/56214477-5ad53980-6034-11e9-9beb-b9ef13a60525.png)

I've downloaded the file from the Microchip Repository, and I managed to compile without further problems.

![ss](https://user-images.githubusercontent.com/17934832/56214774-dafb9f00-6034-11e9-8d16-e09d74b9fbbf.png)

**Comment by eliveltonpsantos** [comment]
You are right: comparing the _uDEV SIGFOX Embedded Development Kit - Manual v. 012_ and my _Device Pack Manager_ is missing the version 1.2.280. Thanks!
